src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const totalCost = days * costPerDay;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days >= 7) {
+    return +(totalCost - bigDiscount);
+  }
+
+  if (days >= 3 && days <= 6) {
+    return +(totalCost - smallDiscount);
+  }
```

Comment:
если ты дошла до этого if, значит days уже точно меньше или равно 6
```suggestion
  if (days >= 3) {
    return +(totalCost - smallDiscount);
  }
```